robocopy: add page (#2953)
tldr-pages/tldr: pages/windows/robocopy.md

# robocopy

> > Robust File and Folder Copy. By default Robocopy will only copy a file if the source and destination have different time stamps or different file sizes.

Reviewer: Perhaps this will read better?
```suggestion
> Robust File and Folder Copy.
> By default files will only be copied if the source and destination have different time stamps or different file sizes.
```

- Copy all .jpg and .bmp files from one directory to another:

> `robocopy {{path/to/source}} {{path/to/destination}} *.jpg *.bmp`

Reviewer 2: Put jpg and bmp within tokens (`{{}}`)

- Copy all files and subdirectories, including empty ones:

`robocopy {{path/to/source}} {{path/to/destination}} /E`

- Mirror/Sync a directory, deleting anything not in source and include all attributes and permissions:

`robocopy {{path/to/source}} {{path/to/destination}} /MIR /COPYALL`

> - Copy all files and subdirectories, excluding older files:

Reviewer 2: Define old. Older than how many days ?

Author: Exclude copying files when source file timestamp is older than destination file timpstamp

`robocopy {{path/to/source}} {{path/to/destination}} /E /XO`

> - List all files (dry run) 50 MBytes or larger in size:

Reviewer 2: What does dry run mean in context of listing files ?

Author: Meaning its a copy dry run... it doesn't copy but only lists the actions

# robocopy

> Robust File and Folder Copy.
> By default files will only be copied if the source and destination have different time stamps or different file sizes.

- Copy all .jpg and .bmp files from one directory to another:

`robocopy {{path/to/source}} {{path/to/destination}} *.jpg *.bmp`

- Copy all files and subdirectories, including empty ones:

`robocopy {{path/to/source}} {{path/to/destination}} /E`

- Mirror/Sync a directory, deleting anything not in source and include all attributes and permissions:

`robocopy {{path/to/source}} {{path/to/destination}} /MIR /COPYALL`

- Copy all files and subdirectories, excluding source files that are older than destination files:

`robocopy {{path/to/source}} {{path/to/destination}} /E /XO`

> - List all files (copy dry run) 50 MBytes or larger in size:

Reviewer 2: "copy dry run" seems a bit odd. How about - "List all files ... in size instead of copying them" ?
